README.rst

```diff
@@ -27,16 +27,16 @@ To install and use deepspeech all you have to do is:
    # Install DeepSpeech
    pip3 install deepspeech
 
-   # Download pre-trained English model and extract
-   curl -LO https://github.com/mozilla/DeepSpeech/releases/download/v0.6.1/deepspeech-0.6.1-models.tar.gz
-   tar xvf deepspeech-0.6.1-models.tar.gz
+   # Download pre-trained English model files
+   curl -LO https://github.com/mozilla/DeepSpeech/releases/download/v0.7.0/deepspeech-0.7.0-models.pbmm
```

Comment:
Advertise TFLite here?

Reply:
This is supposed to be a simple, no decisions, just copy and paste to see how it works set of instructions. In the line above we install `deepspeech`, not `deepspeech-tflite`, so advertising it here would require somewhat extensive rewording. Similar to my other comment, I'm trying to keep changes to a minimum here since this is a version bump PR.